File: Examples/CSharp/AsposePDF/Forms/MoveFormField.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.InteractiveFeatures.Forms;

namespace CSharp.AsposePdf.Forms
{
    public class MoveFormField
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Forms();

            //open document
            Document pdfDocument = new Document(dataDir + "MoveFormField.pdf");

            //get a field
            TextBoxField textBoxField = pdfDocument.Form["textbox1"] as TextBoxField;

            //modify field location
            textBoxField.Rect = new Aspose.Pdf.Rectangle(300, 400, 600, 500);

            //save modified document
            pdfDocument.Save(dataDir + "MoveFormField_out.pdf");

        }
    }
}

File: Examples/VisualBasic/AsposePDF/Forms/MoveFormField.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf.Forms
Imports Aspose.Pdf

Namespace AsposePDF.Forms
    Public Class MoveFormField
        Public Shared Sub Run()
            ' ExStart:MoveFormField
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Forms()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("MoveFormField.pdf"))

            ' Get a field
            Dim textBoxField As TextBoxField = TryCast(pdfDocument.Form("textbox1"), TextBoxField)

            ' Modify field location
            textBoxField.Rect = New Rectangle(300, 400, 600, 500)

            dataDir = dataDir & Convert.ToString("MoveFormField_out_.pdf")
            ' Save modified document
            pdfDocument.Save(dataDir)
            ' ExEnd:MoveFormField
            Console.WriteLine(Convert.ToString(vbLf & "Form field moved successfully to a new location." & vbLf & "File saved at ") & dataDir)

        End Sub
    End Class
End Namespace
